docs/manuals/source/DeveloperGuide/Webui.rst

```diff
@@ -12,22 +12,24 @@ Translation
 Where do I find the translation?
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-The localization files are located in `webui/module/Application/language`. You can find the languages that are currently
+The localization files are located in file:`webui/module/Application/language`.
+You can find the languages that are currently
 in translation in our online project, which is addressed in the next section.
 
 How should I edit translation files?
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-You need to join our online project for translating the Bareos WebUI. We use https://poeditor.com.
-The following link will redirect you to the right project. You are welcome to join our translation
-community as a contributor there.
-
-https://poeditor.com/join/project/ELnLNbvQJb
+You are welcome to join our translation community via https://poeditor.com/join/project/ELnLNbvQJb.
+After selecting the languages you want to translate,
+you can add and modify translation strings.
+You can also leave comments, in case there is need for further discussions.
```

Comment:
```suggestion
You can also leave comments, in case there is need for further discussions and mark those as fuzzy.
```

Reply:
I reformulated the sentence and add the fuzzy remark.